src/alto-ui/Dropdown/Dropdown.scss

```diff
@@ -43,13 +43,14 @@
 }
 
 .Dropdown__wrapper-trigger {
+  max-width: inherit;
   &--clearable {
     position: relative;
   }
 }
 
 .Dropdown__trigger {
-  max-width: 100%;
+  max-width: inherit;
```

Comment:
remove the property instead no ?

Reply:
then the issue isn't solve and the very long content goes out of the Dropdown__trigger